static/src/javascripts/projects/common/modules/ui/subscription-banner.js

```diff
@@ -55,6 +57,30 @@ const hideBannerInTheseRegions: ReaderRevenueRegion[] = [
 const subscriptionUrl = `${subscriptionHostname}/subscribe/digital?INTCMP=gdnwb_copts_banner_subscribe_SubscriptionBanner&acquisitionData=%7B%22source%22%3A%22GUARDIAN_WEB%22%2C%22campaignCode%22%3A%22subscriptions_banner%22%2C%22componentType%22%3A%22${COMPONENT_TYPE}%22%2C%22componentId%22%3A%22${OPHAN_EVENT_ID}%22%7D`;
 const signInUrl = `${signinHostname}/signin?utm_source=gdnwb&utm_medium=banner&utm_campaign=SubsBanner_Exisiting&CMP_TU=mrtn&CMP_BUNIT=subs`;
 
+const hasAcknowledged = bannerRedeploymentDate => {
+    const redeploymentDate = new Date(bannerRedeploymentDate).getTime();
+    const lastClosedAt = userPrefs.get(SUBSCRIPTION_BANNER_CLOSED_KEY);
+    const lastClosedAtTime = new Date(lastClosedAt).getTime();
+
+    return lastClosedAt && lastClosedAtTime > redeploymentDate;
+};
+
+const hasAcknowledgedBanner = region =>
+    fetchJson(`/reader-revenue/subscriptions-banner-deploy-log/${region}`, {
+        mode: 'cors',
+    })
+        .then(resp => hasAcknowledged(resp.time))
+        .catch(err => {
+            reportError(
+                new Error(
+                    `Unable to get subscriptions banner deploy log: ${err}`
+                ),
+                { feature: 'subscriptions-banner' },
+                false
+            );
+            return false;
```

Comment:
maybe this should be `return true` because then we'd default to not showing the banner when these requests fail?